### ISSUE TITLE
Allow creating new project via keyboard

### ISSUE THREAD

**Issue by nidarasheed** [toggl-open-source/toggldesktop#3840]
### 💻 Environment

Platform:  MacOS
OS Version:  10.15.3
Toggl Version:  7.5.42

### 🐞 Actual behavior

User cannot add projects via keyboard only. The Enter button does nothing,  you have to click Add button. 

### 💯 Expected behavior

User should be able to do this with just a keyboard. 

### 📦 Additional info

[Case reference](https://app.intercom.io/a/apps/ayixs927/inbox/inbox/534741/conversations/25829407894)
